### ISSUE TITLE
brl.appstub fail to compile threaded (atleast) on OpenSuse & Windows 8.1

### ISSUE THREAD

**Issue by olejr** [bmx-ng/brl.mod#9]
bmk makemods -h / bmk makemods -g x64 -h gives:

Compile Error: Illegal type expression 
[/brl.mod/appstub.mod/debugger_mt.stdio.bmx;422;0]

Easy fix, just a missing type on function parameter.

But it also throws a Conflicting types for 'bbThreadGetData'.
Don't know if that's the module, or bmk/bcc bug, as some of the lines listed are from the /.bmx folder.

**Comment by GWRon** [comment]
You are right: line 422 is this:

```BlitzMax
Function bbThreadSetData( index,data:Object )
```
but should be
```BlitzMax
Function bbThreadSetData( index:int,data:Object )
```
to be `SuperStrict`.

@ conflicting types:
I think there need to be some more lines in `debugger_mt.stdio.x` (this file contains functions declared before) but I am not sure on this - also I do not know how to define the return-type in this file. Think Brucey will fix this within seconds.

**Comment by GWRon** [comment]
For now I opened __brl.mod/appstub.mod/debugger_mt.stdio.x__ and appended the following line `BBObject* bbThreadGetData( size_t )!`

so it looked like this:
```
void bbOnDebugStop()!
void bbOnDebugLog( BBString * )!
void bbOnDebugEnterStm( BBDebugStm * )!
void bbOnDebugEnterScope( BBDebugScope * )!
void bbOnDebugLeaveScope()!
void bbOnDebugPushExState()!
void bbOnDebugPopExState()!
void bbOnDebugUnhandledEx( BBObject * )!
int   bbGCValidate( void * )!
BBObject* bbThreadGetData( size_t )!
```

It compiled but I did not test it. I am also not sure if this this correct, because "bbThreadGetData" is defined in two different ways in BlitzMax:

threads.mod/threads.bmx: `Function bbThreadGetData:Object( index:Int )`
appstub.mod/debugger_mt.stdio.bmx: `Function bbThreadGetData:TDbgState( index:Int )`

So it might be done in another way. But like said. Brucey will surely have a fix for this.
